test/test_logging.py

```diff
@@ -40,23 +41,6 @@ def test_skip_logging_configuration(
         # Nothing should be captured by capfd since structlog is not configured
         assert capfd.readouterr().err == ""
 
-    def test_skip_logging_if_structlog_not_installed(
```

Comment:
this somehow breaks pytest itself. I tried to  fix it but couldn't figure out what happened. Would be nice to test the import error but in my opinion also not super crucial (it's hard to test in the first place)